packages/perseus/src/__tests__/renderer-api_test.jsx

```diff
@@ -149,30 +149,6 @@ describe("Perseus API", function () {
     });
 
     describe("CSS ClassNames", function () {
-        describe("perseus-input", function () {
```

Comment:
I think we're ok. Those styles are actually things we don't want because wonderblocks is doing the styling now. (I'm 90% sure but I think robert doing regression testing will help make sure 100%)